packages/block-library/src/columns/style.scss

```diff
@@ -10,41 +10,37 @@
 }
 
 .wp-block-column {
-	flex: 1;
+	flex-grow: 1;
```

Comment:
I don't think this property is necessary since we are setting the width to 100% below.